jenkins-jobs/scripts/deploy-snapshots.groovy

```diff
@@ -0,0 +1,58 @@
+import groovy.json.*
+import java.util.stream.*
+
+if (
+    !DEBEZIUM_REPOSITORY ||
+    !DEBEZIUM_BRANCH ||
+    !DEBEZIUM_INCUBATOR_REPOSITORY ||
+    !DEBEZIUM_INCUBATOR_BRANCH
+) {
+    error 'Input parameters not provided'
+}
+
+GIT_CREDENTIALS_ID = '17e7a907-8401-4b7e-a91b-a7823047b3e5'
```

Comment:
Is this something secret?

Reply:
@gunnarmorling It is a handle to the Jenkins credentials and secrets store